client/src/lib/fetch.js

```diff
@@ -1,10 +1,18 @@
 import { IN_PERSON, REMOTE, SERVER_URL } from "../constants";
 
-export async function getUserData(id) {
-  const getUserDataEndpoint = `${SERVER_URL}/user/${id}`;
-  const res = await fetch(getUserDataEndpoint, {
-    credentials: "include",
-  });
+export async function getUserData(id = null) {
```

Comment:
I did the same thing in my PR but in reverse, using `id="me"` as default.

Reply:
Oh yeah for sure - I actually think I like your implementation better, so when it comes to merging we can keep yours